app/models/buck_wad.rb

```diff
@@ -20,9 +26,12 @@ class BuckWad
   end
   def initialize(to: '', from: '', for_message: '', count: 1, dept: '')
     @from = T.let(from.to_s, String)
+    @to = T.let(to.to_s, String)
     @for_message = T.let(for_message.to_s, String)
+    @dept = dept
```

Comment:
Should this be `||=` in case there's no value for dept?
```suggestion
    @dept ||= dept
```

Reply:
We don't need that because instance variables are nil by default anyway.